### ISSUE TITLE
Add known limitations to the accessibility statement

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3487]
Also add info on how to access the data in a way that's accessible.

**Comment by eelzi-ONRR** [comment]
Starting to add known limitations: 

Data visualizations are not fully compliant - however all data shown in the visualizations is in the downloadable compliant files.

Some of the filter options are not keyboard accessible. 

If the user needs filtered data and cannot access those functions they can contact us with their request

**Comment by cthomasONRR** [comment]
These updates are in PR https://github.com/DOI-ONRR/nrrd/pull/3496
